2-GenSoftware.sh

```diff
@@ -88,7 +88,9 @@ mkdir -p ~/.config/autostart
 cp ./config_files/tilda.desktop ~/.config/autostart
 cp ./config_files/redshift-gtk.desktop ~/.config/autostart
 
-execute sudo apt-get install htop gparted expect -y
+execute sudo apt-get install htop indicator-cpufreq gparted expect -y
+execute cpufreq-info
```

Comment:
This command doesn't exist. Maybe `indicator-cpufreq`?
But that just starts the applet, not sure why to run in terminal